azure-kusto-ingest/azure/kusto/ingest/V2/compression_type.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class CompressionType(Enum):
+    Uncompressed = ("Uncompressed",)
```

Comment:
why do you need the strings?

Reply:
in Python each member of an enum needs to have a value, i can change it to 
 Uncompressed = auto() if you think thats better